lib/superstring/dynamic-variables.js

```diff
@@ -312,7 +312,7 @@ var faker = require('faker/locale/en'),
             generator: faker.finance.mask
         },
         $randomPrice: {
-            description: 'A random price between 100.00 and 999.00',
+            description: 'A random price between 0.00 and 999.00',
```

Comment:
@coditva The correct value here is actually `between 0.00 and 1000.00` - both ends of the range were incorrect

Reply:
@neilstudd Thanks for pointing out! I missed this. 🙈 

Fix in #1209